packages/proxy/lib/http/index.ts

```diff
@@ -207,6 +214,7 @@ export function _runStage (type: HttpStages, ctx: any, onError: Function) {
       try {
         middleware.call(fullCtx)
       } catch (err) {
+        errorUtils.logError(err)
```

Comment:
This does seem a bit heavy handed - how often does this actually contain actionable info?

OTOH: it's better than just silently (or weirdly) failing. I could go either way.